profiler/src/Demos/Samples.Computer01/Properties/launchSettings.json

```
@@ -1,23 +1,29 @@
 {
   "profiles": {
-    "Computer01": {
+    "Computer01-Alone": {
+      "commandName": "Executable",
+      "executablePath": "$(BaseOutputPath)\\$(ConfigBasedRelativeOutputPath)\\profiler\\src\\Demos\\Samples.Computer01\\$(TargetFramework)\\Samples.Computer01.exe",
+      "nativeDebugging": true
+    },
+    "Computer01+Profiler": {
       "commandName": "Executable",
       "executablePath": "$(BaseOutputPath)\\$(ConfigBasedRelativeOutputPath)\\profiler\\src\\Demos\\Samples.Computer01\\$(TargetFramework)\\Samples.Computer01.exe",
       "environmentVariables": {
         "COR_ENABLE_PROFILING": "1",
+        "COR_PROFILER": "{BD1A650D-AC5D-4896-B64F-D6FA25D6B26A}",
         "COR_PROFILER_PATH_64": "$(BuildOutputRoot)\\bin\\$(Configuration)-x64\\profiler\\src\\ProfilerEngine\\Datadog.Profiler.Native.Windows\\Datadog.Profiler.Native.dll",
         "COR_PROFILER_PATH_32": "$(BuildOutputRoot)\\bin\\$(Configuration)-x86\\profiler\\src\\ProfilerEngine\\Datadog.Profiler.Native.Windows\\Datadog.Profiler.Native.dll",
-        "COR_PROFILER": "{BD1A650D-AC5D-4896-B64F-D6FA25D6B26A}",
 
         "CORECLR_ENABLE_PROFILING": "1",
+        "CORECLR_PROFILER": "{BD1A650D-AC5D-4896-B64F-D6FA25D6B26A}",
         "CORECLR_PROFILER_PATH_64": "$(BuildOutputRoot)\\bin\\$(Configuration)-x64\\profiler\\src\\ProfilerEngine\\Datadog.Profiler.Native.Windows\\Datadog.Profiler.Native.dll",
         "CORECLR_PROFILER_PATH_32": "$(BuildOutputRoot)\\bin\\$(Configuration)-x86\\profiler\\src\\ProfilerEngine\\Datadog.Profiler.Native.Windows\\Datadog.Profiler.Native.dll",
-        "CORECLR_PROFILER": "{BD1A650D-AC5D-4896-B64F-D6FA25D6B26A}",
 
         "COMPlus_EnableDiagnostics": "1",
 
         "DD_PROFILING_ENABLED": "1",
-        "DD_DOTNET_PROFILER_HOME": "$(BuildOutputRoot)\\bin\\$(Configuration)-AnyCPU\\profiler\\src\\ProfilerEngine\\Datadog.Profiler.Managed\\",
+        "DD_PROFILING_CPU_ENABLED": "1",
+        "DD_PROFILING_EXCEPTION_ENABLED": "1",
```

Was this intentionally missing?

```suggestion
        "DD_PROFILING_EXCEPTION_ENABLED": "1",
        "DD_PROFILING_ALLOCATION_ENABLED": "1",
```

---

Yes it is  :^)
Allocation profiling is not yet available and I only updated demo apps that allocate memory (BuggyBits and Computer01)